src/event/mod.rs

```diff
@@ -23,9 +24,7 @@ pub mod proto {
 }
 
 lazy_static! {
-    pub static ref MESSAGE: Atom = Atom::from("message");
-    pub static ref HOST: Atom = Atom::from("host");
-    pub static ref TIMESTAMP: Atom = Atom::from("timestamp");
+    pub static ref LOG_SCHEMA: OnceCell<LogSchema> = OnceCell::new();
```

Comment:
Should this be in config?

Reply:
This doesn't need to be a lazy_static https://docs.rs/once_cell/1.3.1/once_cell/sync/struct.OnceCell.html#method.new since new is a const fn :)